django/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.12.4-slim
+
+WORKDIR /app
+
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED=1
+
```

Comment:
Missing `apt update && apt upgrade -y` and `python3 -m pip install --upgrade pip`

Reply:
why update apt ? we are not using any package from it